src/dwdownloader.py

```diff
@@ -136,12 +136,11 @@ def download(url):
 
 def change_wallpaper():
     config = Configuration()
-    source = config.get('source')
-    random = config.get('random')
-    if random:
-        modules = get_modules()
-        selected = randrange(len(modules))
-        source = modules[selected]
+    if config.get('random'):
+        sources = config.get('source')
+        source = sources[randrange(len(sources))]
+    else:
+        source = config.get('source')[0]
```

Comment:
If random mode is not selected, the first source is used from the list of sources saved in the configuration.
This makes it easier to handle the case when the configuration file is manipulated externally for some reason, by hand, for example. But during normal operation this should never happen as the UI allows selection of multiple sources only in random mode.